lib/roo_on_rails/checks/heroku/app_exists.rb

```diff
@@ -16,6 +16,10 @@ module Heroku
       # Output context:
       # - heroku.app.{env}: an app name.
       class AppExists < EnvSpecific
+        ACCEPTABLE_ENV_NAMES = {
+          'production' => ['prod', 'production'],
+          'staging'    => ['stg', 'staging']
```

Comment:
did we reach any consensus on this? i'd suggest using 'test' or 'stag' instead of 'stg' so prod/test and prod/stag are both 4 chars long